### PR TITLE
Reload group

### PR DESCRIPTION
**Pull request by edoshor**
Add a button to context menu to reload a specific group

### CODE REVIEW

File: frontend/src/app/user/user.controller.ts [hunk 1]
```diff
@@ -27,17 +27,33 @@ export class UserController {
...
+          this.reload();
+        }
+        break;
     }
+
+  }
+
+  reload() {
+    this.toastr.info('Admin asked us to reload');
+    window.location.reload();
```
Comment (kkavod) [inline frontend/src/app/user/user.controller.ts]: use $location service instead

**Comment by kkavod** [comment]
LGTM, please merge

File: frontend/src/app/user/user.controller.ts [hunk 2]
```diff
@@ -27,17 +27,33 @@ export class UserController {
   }
 
   onMessage(message: any) {
-    if (message.message === 'toggleAudio') {
-      this.videoRoom.toggleLocalAudio(message.enabled);
+    switch (message.message) {
+      case 'toggleAudio':
+        this.videoRoom.toggleLocalAudio(message.enabled);
+        break;
+      case 'reload':
```
Comment (kolmanv) [inline frontend/src/app/user/user.controller.ts]: Why do we need to handle 'reload' in onMessage? Why not only in onAdminMessage?

**Comment by kolmanv** [comment]
LGTM, you can merge.

File: frontend/src/app/user/user.controller.ts (hunk 2 above)
Comment (edoshor) [inline frontend/src/app/user/user.controller.ts]: So we could send the message to /user/login

**Pull request merged**